modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -77,6 +77,9 @@ ComputeMultiPlasticityStress::ComputeMultiPlasticityStress(const InputParameters
     _my_elasticity_tensor(RankFourTensor()),
     _my_strain_increment(RankTwoTensor())
 {
+  if (_epp_tol <= 0)
+    mooseError("ComputeMultiPlasticityStress: ep_plastic_tolerance must be positive");
+
```

Comment:
yea, when you simply write

params.addRequiredRangeCheckedParam<Real>("andy_brain_size", "andy_brain_size>0", "Should be set enormously huge");

then if someone sets "andy_brain_size=1E-12" it violates the RangeChecking, as "zero" in the above check is actually "1E-12".